### ISSUE TITLE
Feat: Make the font size configurable

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#3963]
## Feature description and goals

The font size is currently hard-coded.

To do:
 - [ ] use the 3-4 font size constants everywhere (we will need to change some font sizes a bit in the UI)
 - [ ] make these constants configurable
 
We will probably need to make these constants use css variables. These variables will be set by the backend in index.mak.

## See also

#3479 - should be done at the same time as this issue
